### PR TITLE
Add reset gpio to max98927 driver

### PR DESCRIPTION
**Pull request by alikates**

### CODE REVIEW

**Comment by SirSireesh** [comment]
LGTM, could you add a Signed-off-by to your commits?

**Comment by alikates** [comment]
Done

File: sound/soc/codecs/max98927.c [hunk 1]
```diff
@@ -898,6 +899,19 @@ static int max98927_i2c_probe(struct i2c_client *i2c,
 		return ret;
 	}
 
+	reset_gpio = devm_gpiod_get_optional(&i2c->dev, "reset", GPIOD_OUT_HIGH);
+	if (IS_ERR(reset_gpio)) {
+		int error = PTR_ERR(reset_gpio);
+		dev_err(&i2c->dev,
+			"Failed to request GPIO reset pin, error %d\n", error);
+		return error;
```
Comment (z3ntu) [inline sound/soc/codecs/max98927.c:907]: You could reuse the ret variable instead of error I guess

**Comment by z3ntu** [comment]
The commit messages need some more detail for upstreaming, not just the subject line.

**Comment by alikates** [comment]
Woah, i think i closed the pr by accident, also squashed both commits

**Comment by alikates** [comment]
I was trying to reopen it, i think it will be easier if i create it again

**Pull request closed without merging**